app/rankers/algorithms.py

```diff
@@ -131,9 +154,9 @@ def thompson_sampling(
     return [x[0] for x in scores]
 
 
-def personalize_topic_slates(input_slate_configs: List['SlateConfigModel'],
-                             personalized_topics: PersonalizedTopicList,
-                             topic_limit: Optional[int] = 1) -> List['SlateConfigModel']:
+def __personalize_topic_slates(input_slate_configs: List['SlateConfigModel'],
```

Comment:
Might be worth putting in the docstring something to the effect of "Instead of calling this directly, make a new public method that calls it (see `top1_topics` for example)." That way the next person who needs this functionality will be at less risk of defaulting to making this method public again and using it directly.